packages/core/src/utils/transformers/media/transformMedia.ts

```diff
@@ -81,6 +89,5 @@ export const transformMedia = (
     return;
   }
 
-  // return getBoundValue(asset, entityStore, binding, path);
-  return getBoundValue(asset, path);
+  return asset.fields.file?.url;
```

Comment:
Also I noticed that it may provide better semantic meaning if we explicitly state the return type of the `transformMedia()` function. 

This helps to explicitly see this that both:
 - magical `OptimizedImageAsset` type is allowed
 - and pedestrian `string` 
 - (and of course undefined in case there's some issue)

<img width="894" alt="image" src="https://github.com/contentful/experience-builder/assets/113918657/62885383-286f-4049-a16d-92e9863fa494">


You may need to wrangle types for the final return statement

```
  return asset.fields.file?.url ? asset.fields.file.url as string : undefined ;
```